documentation/src/main/asciidoc/user_guide/jcache.adoc

```diff
@@ -59,6 +59,26 @@ Cache<String, String> cache = cacheManager.createCache("namedCache",
 ----
 
 
+=== Create a remote cache
+Creating a remote cache (client-server mode), using default configuration options
+as defined by the JCache API specification, is as simple as doing the following:
+
+[source,java]
+----
+import javax.cache.*;
+import javax.cache.configuration.*;
+
+// Retrieve the system wide cache manager via org.infinispan.jcache.remote.JCachingProvider
+CacheManager cacheManager = Caching.getCachingProvider("org.infinispan.jcache.remote.JCachingProvider").getCacheManager();
+// Define a named cache with default JCache configuration
+Cache<String, String> cache = cacheManager.createCache("remoteNamedCache",
+      new MutableConfiguration<String, String>());
+----
+
+NOTE: In order to use the org.infinispan.jcache.remote.JCachingProvider, the
+infinispan-jcache-remote-<version>.jar needs to be on put your classpath.
```

Comment:
Hi Slaskawi,

Thanks for the comment.  But I'm wondering if the `infinispan-remote.jar` is really required.   Below is the `pom.xml` for my test, and it works as expected.

```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>
	<groupId>com.cf.spike</groupId>
	<artifactId>infinispan-client</artifactId>
	<version>1.0.1-SNAPSHOT</version>
	<dependencies>
		<dependency>
			<groupId>org.infinispan</groupId>
			<artifactId>infinispan-jcache-remote</artifactId>
			<version>7.2.5.Final</version>
		</dependency>
		<dependency>
			<groupId>org.wildfly</groupId>
			<artifactId>wildfly-controller</artifactId>
			<version>8.1.0.Final</version>
		</dependency>
	</dependencies>
</project>
```